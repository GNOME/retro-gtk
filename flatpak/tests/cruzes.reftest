[Retro Reftest]
Path=/cruzes
Core=/app/lib/libretro/cruzes_libretro.so

[Options]

[Frame 0]
State=None
Video=cruzes.png
