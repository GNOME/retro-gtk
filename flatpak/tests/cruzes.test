[Retro Reftest]
Path=/cruzes
Core=/app/lib/libretro/cruzes_libretro.so

[Frame 0]
Tests=Run;Video;NoStateAccess;
Video=cruzes.png
