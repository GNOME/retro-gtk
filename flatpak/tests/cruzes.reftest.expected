/cruzes/Boot: OK
/cruzes/Options: OK
/cruzes/0/State None: OK
/cruzes/0/Run: OK
/cruzes/0/Video: OK
